taskcluster/ci/config.yml

```diff
@@ -0,0 +1,29 @@
+---
```

Comment:
This file is a copy of https://github.com/mozilla-mobile/fenix/blob/34144fd254f00bd8a4176a296fad384ced69f2bf/taskcluster/ci/config.yml.

I removed workers that won't be used (signing, beetmover, push-apk) and renamed Fenix references.

Reply:
++ did the same in application-services so I'm happy to see I'm on the right path!